Scripts/SDL/EderNaybree.sdl

```diff
@@ -0,0 +1,27 @@
+STATEDESC EderNaybree
```

Comment:
Please add the GPL license block as found in the other SDL files. Check this as an example: https://github.com/H-uru/Plasma/blob/master/Scripts/SDL/BaronCityOffice.sdl#L1:L41